diff --git a/node_modules/playroom/lib/makeWebpackConfig.js b/node_modules/playroom/lib/makeWebpackConfig.js
index 6ca8355..9362b87 100644
--- a/node_modules/playroom/lib/makeWebpackConfig.js
+++ b/node_modules/playroom/lib/makeWebpackConfig.js
@@ -138,7 +138,7 @@ module.exports = async (playroomConfig, options) => {
         },
         {
           test: /\.css$/,
-          include: path.dirname(require.resolve('codemirror/package.json')),
+          include: [path.dirname(require.resolve('codemirror/package.json'))],
           use: [MiniCssExtractPlugin.loader, require.resolve('css-loader')],
         },
       ],
