diff --git a/node_modules/storybook-addon-playroom/dist/manager.js b/node_modules/storybook-addon-playroom/dist/manager.js
index 21ac58f..d59539f 100644
--- a/node_modules/storybook-addon-playroom/dist/manager.js
+++ b/node_modules/storybook-addon-playroom/dist/manager.js
@@ -8,6 +8,6 @@ function _interopDefault (e) { return e && e.__esModule ? e : { default: e }; }
 
 var m__default = /*#__PURE__*/_interopDefault(m);
 
-var r="storybook/playroom",e=`${r}/tab`,a="playroom",s={UPDATE:`${r}/update`};var f=theming.styled.p({textAlign:"center"}),A=theming.styled.iframe({border:"0 none",height:"100%",width:"100%"}),p=({active:o})=>{let[t,i]=m.useState("");return managerApi.useChannel({[s.UPDATE]:i}),o?t?m__default.default.createElement(A,{key:t,allowFullScreen:!0,src:t,title:"Playroom"}):m__default.default.createElement(f,null,"Playroom has been disabled for this story."):null};managerApi.addons.register(r,()=>{managerApi.addons.add(e,{type:managerApi.types.TAB,title:"Playroom",route:({storyId:o})=>`/playroom/${o}`,match:({viewMode:o})=>o==="playroom",render:p,paramKey:a});});
+var r="storybook/playroom",e=`${r}/tab`,a="playroom",s={UPDATE:`${r}/update`};var f=theming.styled.p({textAlign:"center"}),A=theming.styled.iframe({border:"0 none",height:"100%",width:"100%"}),p=({active:o})=>{let[t,i]=m.useState("");return managerApi.useChannel({[s.UPDATE]:i}),o?t?m__default.default.createElement(A,{key:t,allowFullScreen:!0,src:t,title:"Playroom"}):m__default.default.createElement(f,null,"Playroom has been disabled for this story."):null};managerApi.addons.register(r,()=>{managerApi.addons.add(e,{type:managerApi.types.PANEL,title:"Playroom",route:({storyId:r})=>`/playroom/${r}`,render:p,paramKey:a});});
 //# sourceMappingURL=out.js.map
 //# sourceMappingURL=manager.js.map
\ No newline at end of file
diff --git a/node_modules/storybook-addon-playroom/dist/manager.mjs b/node_modules/storybook-addon-playroom/dist/manager.mjs
index aff2d19..7a4b16b 100644
--- a/node_modules/storybook-addon-playroom/dist/manager.mjs
+++ b/node_modules/storybook-addon-playroom/dist/manager.mjs
@@ -2,6 +2,6 @@ import { addons, types, useChannel } from '@storybook/manager-api';
 import { styled } from '@storybook/theming';
 import m, { useState } from 'react';
 
-var r="storybook/playroom",e=`${r}/tab`,a="playroom",s={UPDATE:`${r}/update`};var f=styled.p({textAlign:"center"}),A=styled.iframe({border:"0 none",height:"100%",width:"100%"}),p=({active:o})=>{let[t,i]=useState("");return useChannel({[s.UPDATE]:i}),o?t?m.createElement(A,{key:t,allowFullScreen:!0,src:t,title:"Playroom"}):m.createElement(f,null,"Playroom has been disabled for this story."):null};addons.register(r,()=>{addons.add(e,{type:types.TAB,title:"Playroom",route:({storyId:o})=>`/playroom/${o}`,match:({viewMode:o})=>o==="playroom",render:p,paramKey:a});});
+var r="storybook/playroom",e=`${r}/tab`,a="playroom",s={UPDATE:`${r}/update`};var f=styled.p({textAlign:"center"}),A=styled.iframe({border:"0 none",height:"100%",width:"100%"}),p=({active:o})=>{let[t,i]=useState("");return useChannel({[s.UPDATE]:i}),o?t?m.createElement(A,{key:t,allowFullScreen:!0,src:t,title:"Playroom"}):m.createElement(f,null,"Playroom has been disabled for this story."):null};addons.register(r,()=>{addons.add(e,{type:types.PANEL,title:"Playroom",route:({storyId:o})=>`/playroom/${o}`,render:p,paramKey:a});});
 //# sourceMappingURL=out.js.map
 //# sourceMappingURL=manager.mjs.map
\ No newline at end of file
diff --git a/node_modules/storybook-addon-playroom/dist/preview.mjs b/node_modules/storybook-addon-playroom/dist/preview.mjs
index fd0b124..e8fe3cc 100644
--- a/node_modules/storybook-addon-playroom/dist/preview.mjs
+++ b/node_modules/storybook-addon-playroom/dist/preview.mjs
@@ -1,8 +1,9 @@
 import { useChannel } from '@storybook/preview-api';
 import { createUrl } from 'playroom/utils';
 import R from 'react-element-to-jsx-string';
+import { unpackDecorator } from "@kickstartds/core/lib/storybook";
 
-var p="storybook/playroom",t="playroom",a={UPDATE:`${p}/update`};var m=({url:o="http://localhost:9000",code:e="",disable:r=!1,reactElementToJSXStringOptions:n={sortProps:!1}}={})=>({url:o,code:e,disable:r,reactElementToJSXStringOptions:n});var c=(o,e)=>{let{parameters:r}=e,n=r[t],{url:s,code:l,reactElementToJSXStringOptions:y}=m(n),i=o(),S=l||R(i,y),f=s&&createUrl({baseUrl:s,code:S});return useChannel({})(a.UPDATE,f),i};var g={decorators:[c],globals:{[t]:!1}},_=g;
+var p="storybook/playroom",t="playroom",a={UPDATE:`${p}/update`};var m=({url:o="http://localhost:9000",code:e="",disable:r=!1,reactElementToJSXStringOptions:n={sortProps:!1}}={})=>({url:o,code:e,disable:r,reactElementToJSXStringOptions:n});var c=(o,e)=>{let{parameters:r, undecoratedStoryFn}=e,n=r[t],{url:s,code:l,reactElementToJSXStringOptions:y,disable}=m(n),i=unpackDecorator(undecoratedStoryFn,e),S=(typeof l !== "function" && l)||R(i,y),f=s&&createUrl({baseUrl:s,code:S});return useChannel({})(a.UPDATE,disable?'':f),o()};var g={decorators:[c],globals:{[t]:!1}},_=g;
 
 export { _ as default };
 //# sourceMappingURL=out.js.map
